include/my_global.h

```diff
@@ -281,13 +281,24 @@ typedef socket_len_t SOCKET_SIZE_TYPE; /* Used by NDB */
 #define FN_NETWORK_DRIVES	/* Uses \\ to indicate network drives */
 #else
 #define FN_LIBCHAR	'/'
-#define FN_LIBCHAR2	'/'
+/*
+  FN_LIBCHAR2 is not defined on !Windows. Use is_directory_separator().
+*/
 #define FN_DIRSEP       "/"     /* Valid directory separators */
 #define FN_EXEEXT   ""
 #define FN_SOEXT    ".so"
 #define FN_ROOTDIR	"/"
 #endif
 
+static inline int is_directory_separator(char c)
+{
+#ifdef _WIN32
+  return c == FN_LIBCHAR || c == FN_LIBCHAR2;
```

Comment:
I am just wondering if this was tested on windows. FN_LIBCHAR2 was removed from line 284 and I cannot find FN_LIBCHAR2 in msdn. Thus I assume this will not compile on windows.